.travis.yml

```diff
@@ -26,4 +26,5 @@ env:
 
 install: beaver dlang install
 
-script: beaver dlang make
+# Limit memory usage on travis with -j1
```

Comment:
Just to avoid confusing casual readers, I would explain a bit more:
```yaml
# DMD can be very memory hungry, and beaver uses make
# -j2 by default, which means 2 DMD will run in parallel, which
# can exhaust Travis VM memory. Because of this we override
# this with -j1 to make sure only one DMD is ran at a time.
```